src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
```

Comment:
But try to make up more descriptive names, not just `value`

Reply:
Sure